### PR TITLE
[CLOUD-3448] typo fix: clearing the correct datasource environmental variables

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.jboss.org/browse/CLOUD-3448

@luck3y @kabir please, take a look

### CODE REVIEW

File: jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh [hunk 1]
```diff
@@ -954,7 +954,7 @@ function inject_datasource() {
 
   if [ -z "$jta" ]; then
     log_warning "JTA flag not set, defaulting to true for datasource  ${service_name}"
-    jta=false
```
Comment (jfdenise) [inline jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh]: @ochaloup  this change is unrelated to the issue. @kabir is it ok from your point of view?

Comment (kabir) [reply jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh]: I think so, although I remember @yersan discussing this - let's wait for him to comment

File: jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh [hunk 2]
```diff
@@ -31,8 +31,8 @@ function clearDatasourceEnv() {
   local prefix=$1
   local service=$2
 
-  unset ${service}_HOST
-  unset ${service}_PORT
+  unset ${service}_SERVICE_HOST
+  unset ${service}_SERVICE_PORT
```
Comment (yersan) [inline jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh]: The change should be harmless, those variables are set by Kubernetes when the services are configured by the templates. In general, I think the env files would expect to see them available. However, I've just seen the same pattern in [other files](https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/messaging/added/launch/messaging.sh#L37) being in use and seems to be the initial intention of clearing ${service}_PORT and ${service}_HOST. So, it looks good to me.

File: jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh (hunk 1 above)
Comment (yersan) [reply jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh]: @kabir I discussed with Ondra only the details about the variables. Regarding the JTA change, assuming JTA by default to true is pretty reasonable in Java EE world, also it was the original intention regarding the logged message. This seems like a bug never reported.

Comment (jfdenise) [reply jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh]: @ochaloup, we should separate this fix I think. Create a new CLOUD issue and a dedicated commit.

Comment (ochaloup) [reply jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh]: @jfdenise sure, I see. I updated the PR and created a new one for this change: https://github.com/wildfly/wildfly-cekit-modules/pull/140

**Comment by ochaloup** [comment]
@jfdenise @yersan I updated the PR
